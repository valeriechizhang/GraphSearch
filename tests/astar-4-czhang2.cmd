--energy 50 --start-x=3 --start-y=3 --end-x=1 --end-y=1