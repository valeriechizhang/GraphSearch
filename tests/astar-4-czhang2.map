26  25  29  15  18  5   27  9   2   2   19  26
1   18  24  20  24  26  4   4   21  8   6   27
2   8   27  7   18  4   27  24  10  15  14  15
13  8   1   9   20  1   16  27  4   21  6   10
16  10  24  4   13  11  6   8   24  16  11  16
13  27  11  11  18  25  2   14  20  9   9   16
20  2   16  19  15  13  19  3   6   11  21  19
26  2   14  27  25  18  22  7   24  23  12  25
4   18  2   4   14  9   29  3   11  10  8   27