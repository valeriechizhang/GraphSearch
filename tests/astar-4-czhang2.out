Solution steps: ['W', 'S', 'S', 'W']
Solution cost: 43
Number of states considered: 7

Frontier:
Pos=(4, 3) Moves=['E'] Cost=50
Pos=(3, 2) Moves=['W', 'S', 'E'] Cost=37
Pos=(3, 5) Moves=['N', 'N'] Cost=34
Pos=(2, 0) Moves=['W', 'S', 'S', 'S'] Cost=43

Closed List:
Pos=(3, 3) Moves=[] Cost=0
Pos=(2, 3) Moves=['W'] Cost=1
Pos=(3, 4) Moves=['N'] Cost=8
Pos=(2, 2) Moves=['W', 'S'] Cost=27
Pos=(2, 1) Moves=['W', 'S', 'S'] Cost=30
Pos=(1, 2) Moves=['W', 'S', 'W'] Cost=42
Pos=(1, 1) Moves=['W', 'S', 'S', 'W'] Cost=43
